app/src/main/java/com/kks/nimblesurveyjetpackcompose/viewmodel/home/HomeViewModel.kt

```diff
@@ -105,20 +100,23 @@ class HomeViewModel @Inject constructor(
         }
     }
 
-    private fun getSurveyList() {
-        if (_currentPageNumber.value <= totalNumberOfPage) {
+    private fun getSurveyList(isClearCache: Boolean = false) {
+        val pageNumber = ((if (isClearCache) START_SURVEY_NUMBER else _surveyList.value.size) / DEFAULT_PAGE_SIZE) + 1
+        if ((pageNumber <= _pages && _surveyList.value.size < _records) || isClearCache) {
```

Comment:
`_pages` doesn't seem too clear in terms of intent 🤔 How about `_pageCount` or `_pageTotal` or `_numberOfPages` instead? 🤔 
Same with `pageNumber`, tho lower priority 🤣 How about `currentPage`? 🤔